even_fib_nums.js

```diff
@@ -7,10 +8,22 @@
  */
 function _sumFibs( maxFibValue ) {
   var sum = 0;
-
+  let newSum = 0;
+  let a = 0;
```

Comment:
Use variable names that help clarify what they are. For example
`fibMinusOne` would be a better name than `b`.
`i` is kind of the one special case as an interator.